cpu/cc26xx_cc13xx/include/cc26xx_cc13xx_uart.h

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @addtogroup      cpu_cc13x2_definitions
+ * @addtogroup      cpu_cc26x2_cc13x2_definitions
```

Comment:
Shouldn't this be `cpu_cc26xx_cc13xx_definitions`?

Reply:
So this must have been leftover from a previous move.